packages/rich-text/src/plugins/EmbeddedEntityInline/FetchingWrappedInlineEntryCard.tsx

```diff
@@ -21,6 +21,83 @@ const styles = {
   }),
 };
 
+type InternalFetchingWrappedInlineEntryCardProps = Pick<
+  FetchingWrappedInlineEntryCardProps,
+  'onEdit' | 'onRemove' | 'isDisabled' | 'isSelected'
+> & {
+  locale: string;
+  defaultLocale: string;
+  entry: Entry;
+  allContentTypes: ReturnType<FieldAppSDK['space']['getCachedContentTypes']>;
+  getEntityScheduledActions: React.ComponentProps<
+    typeof ScheduledIconWithTooltip
+  >['getEntityScheduledActions'];
+  entryStatus: ReturnType<typeof getEntryStatus>;
+};
+
+function InternalFetchingWrappedInlineEntryCard({
+  entry,
+  allContentTypes,
+  locale,
+  defaultLocale,
+  isSelected,
+  entryStatus,
+  getEntityScheduledActions,
+  onEdit,
+  onRemove,
+  isDisabled,
+}: InternalFetchingWrappedInlineEntryCardProps) {
+  const contentType = React.useMemo(() => {
+    if (!allContentTypes) {
+      return undefined;
+    }
+
+    return allContentTypes.find(
+      (contentType) => contentType.sys.id === entry.sys.contentType.sys.id
+    );
+  }, [allContentTypes, entry]);
+
+  const title = React.useMemo(
+    () =>
+      getEntryTitle({
+        entry,
+        contentType,
+        localeCode: locale,
+        defaultLocaleCode: defaultLocale,
+        defaultTitle: 'Untitled',
+      }),
+    [entry, contentType, locale, defaultLocale]
+  );
+
+  const contentTypeName = contentType ? contentType.name : '';
```

Comment:
This feels wrong but can be fixed in a follow (already there in the existing code)

Reply:
I just it to also not show the `:` if the contentType would not be available 👍